### PR TITLE
Add feed experiment for 4/30

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

Config adjustments for feed after seeing success from one of the new variants last week.

## Related Tickets & Documents

- Related Issue https://github.com/forem/forem/issues/19387

## QA Instructions, Screenshots, Recordings

Proofreading should suffice.

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: Only config adjustments, code still covered.
- [ ] I need help with writing tests

### CODE REVIEW

File: config/feed-variants/20230424-variant-b.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "max_days_since_published": 15,
-  "description": "Built off 20230110-variant-c but more weight on tags.",
+  "description": "Built off 20230424-variant-b but with steeper experience level cliff.",
```
Comment (benhalpern) [inline config/feed-variants/20230424-variant-b.json:3]:
```suggestion
  "description": "Built off 20230110-variant-c but more weight on tags.",
```

File: config/feed-variants/20230424-variant-b.json [hunk 2]
```diff
@@ -69,15 +69,15 @@
...
-        [4, 0.77],
-        [5, 0.55],
-        [6, 0.35],
-        [7, 0.3]
+        [2, 0.5],
+        [3, 0.4],
+        [4, 0.3],
+        [5, 0.2],
+        [6, 0.15],
+        [7, 0.13]
```
Comment (benhalpern) [inline config/feed-variants/20230424-variant-b.json:77]:
```suggestion
        [2, 0.95],
        [3, 0.88],
        [4, 0.77],
        [5, 0.55],
        [6, 0.35],
        [7, 0.3]
```

File: config/feed-variants/20230424-variant-b.json [hunk 3]
```diff
@@ -77,7 +77,7 @@
         [7, 0.3]
       ],
       "default_user_experience_level": 5,
-      "fallback": 0.2
+      "fallback": 0.001
```
Comment (benhalpern) [inline config/feed-variants/20230424-variant-b.json:80]:
```suggestion
      "fallback": 0.2
```

**Review by maestromac — APPROVED** [review]
LGTM but is there a doc file I can match all these changes to? Because other than that, i'm not really sure what to review on feed experiment PRs

File: config/feed-variants/20230430-variant-a.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        [3, 0.5],
+        [4, 0.6],
+        [5, 0.7],
+        [6, 0.8],
+        [7, 0.85],
+        [8, 0.9],
+        [9, 0.95],
+        [10, 0.97],
+        [11, 0.98],
+        [12, 0.99]
```
Comment (maestromac) [inline config/feed-variants/20230430-variant-a.json:103]: These lines are intentionally removed on the other two variants?

**Pull request merged**